tests/conftest.py

```diff
@@ -313,7 +313,7 @@ def teardown_subscribe_to_cdn_dogfood():
 def setup_epel_repository(request, ansible_module):
     """Setup/teardown fixture used by test_positive_check_epel_repository
     and test_positive_check_epel_repository_with_invalid_repo"""
-    setup = ansible_module.yum(name=epel_repo, state="present")
+    setup = ansible_module.yum(name=epel_repo, state="present", disable_gpg_check="yes")
```

Comment:
Just curious, why do we require to set the `disable_gpg_check` option for DNF in EL8 to enable the Epel repository?

Reply:
This is likely because of this change [https://github.com/ansible/ansible/pull/71537](url)
Because this change DNF packages are by default gpg-verified.  
Here we have two ways either use 
1. `disable_gpg_check` parameter      OR
2. add `ansible_module.rpm_key(key=epel_key, state="present")` where `epel_key` will be introduced in constants.py

Here, I've used `disable_gpg_check` since saves one line :)